node/sc/sub_bridge_handler.go

```diff
@@ -342,6 +376,39 @@ func (sbh *SubBridgeHandler) LocalChainHeadEvent(block *types.Block) {
 	}
 }
 
+func (sbh *SubBridgeHandler) handleParentChainInvalidTxResponseMsg(msg p2p.Msg) error {
```

Comment:
Can you add simple description what this function does?
Actually yesterday explanation helped me to understand this code, so adding description would be nice.